src/extraction.py

```diff
@@ -168,12 +174,18 @@ def extract_random_windows(path, stepSize, windowSize, windows, xmlDic, text, pl
         for i in range(windows):
             x = random.choice(allowed_x)
             y = random.choice(allowed_y)
-            window = (x, y, img[x:x + windowSize[0], y:y + windowSize[1]])
+            window = (x, y, img[y:y + 32, x:x + 32])
+
+            if window[2].shape != (32, 32, 3):
+                print('-----------------------')
```

Comment:
ist bei der Massendaten-verarbeitung so ne Sache, sollte nicht, wird aber :S wahrscheinlich dann am besten nen try/catch in der aufrufenden Methode drumherum, wollte aber das es definitiv weitergeht - ob mal ein window nicht klappt spielt bei 40000 nicht so die Rolle